index.js

```diff
@@ -175,6 +187,28 @@ const parseArgs = (
   return result;
 };
 
+// Look ahead to next token in argv array:
+function peek(argv, pos) {
+  if (pos + 1 >= argv.length) {
+    return 0;
+  }
+  return token(argv[pos + 1]);
+}
+
+// Guess token type based on individual string from argv:
+function token(arg) {
+  const chr = StringPrototypeCharAt(arg, 0);
+  const nextChr = StringPrototypeCharAt(arg, 1);
+  if (StringPrototypeMatch(chr, /^[0-9]$/)) {
+    return TOKENS.NUMERIC;
+  } else if (chr === '-' && StringPrototypeMatch(nextChr, /^[0-9]$/)) {
```

Comment:
```suggestion
  } else if (chr === '-' && RegExpPrototypeSymbolMatch(/^[0-9]$/, nextChr)) {
```

Reply:
This should also use test

Reply:
Only testing for one digit? So `-1foo` is returning numeric? It does not actually matter too much since to support negative numbers we are dropping support for digits, but I feel misleading to call it NUMERIC based on that test.